cclib/bridge/cclib2horton.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Bridge for using cclib data in Horton (http://theochem.github.io/horton)."""
+
+import numpy
+from cclib.parser.data import ccData
+from cclib.parser.utils import find_package
+
+_found_horton = find_package('horton')
+if _found_horton:
+    try:
+        from horton import __version__
+    except:
+        _old_horton = True
+    if (__version__[0] == '2'):
+        # Horton 2
+        from horton.io.iodata import IOData
+        _old_horton = False
+    
+_found_iodata = find_package('iodata')
+if _found_iodata:
+    from iodata import IOData
+    _old_horton = False
+    # TODO: Add appropriate routines for Horton3 IOData
+    
+def _check_horton(found_horton, found_iodata, old_horton):
+    if (old_horton):
+        raise ImportError("You must have at least version 2 of `horton` to use this function.")
+    elif (not found_horton and not found_iodata):
+        raise ImportError("You must install `horton` to use this function.")
+
+def makehorton(atomcoords = None, atomnos = None, mocoeffs = None, mult = None, coreelectrons = None, atomcharges = None, polarizability = None):
```

Comment:
Would it be better to just pass in a ccData object?

Reply:
@langner I was thinking of doing that (and actually wrote the first version this way) but then found out that all other bridges are implemented this way. I chose to keep it consistent with other bridge methods, but I see that passing in a `ccData` object would likely be cleaner. I am open to making modifications so that `ccData` object is passd instead.

Reply:
I think this might be a good place to not be consistent; IMO passing the object would be better and we just didn't think about it before. We can update the remaining bridges later. @berquist for thoughts

Reply:
Yes, I agree.

Reply:
Actually, this would lead down a path that isn't worthwhile right now, it'd take too much time.